checks.d/vsphere.py

```diff
@@ -748,7 +748,13 @@ def _collect_metrics_atomic(self, instance, mor):
                     continue
                 instance_name = result.id.instance or "none"
                 value = self._transform_value(instance, result.id.counterId, result.value[0])
-                self.gauge(
+
+                # Most common metric types are absolute, delta, and rate
+                if self.metrics_metadata[i_key][result.id.counterId]['s_type'] == 'rate':
```

Comment:
Hmmm, this is an older PR, I may have made a mistake here. Somehow I came to the conclusion that `s_type` was present because of this https://github.com/DataDog/dd-agent/blob/zeller/vsphere-basic/checks/libs/vmware/basic_metrics.py